ble/CallChainOfFunctionPointersWithContext.h

```diff
@@ -120,11 +161,40 @@ class CallChainOfFunctionPointersWithContext {
      *        chained FunctionPointers.
      */
     void call(ContextType context) {
-        if (chainHead) {
-            chainHead->call(context);
+        ((const CallChainOfFunctionPointersWithContext*) this)->call(context);
+    }
+
+    /**
+     * @brief same as above but const 
+     */
+    void call(ContextType context) const {
+        currentCalled = chainHead;
+
+        while(currentCalled) { 
+            currentCalled->call(context);
```

Comment:
No, it will call the const version of the function, unfortunately, we can't get rid of non const version because minar and mbed os does not allow to build a function pointer with a const member function. This defect, will be fixed, in the future